### ISSUE TITLE
SIW.php : aviso "de habilitar os pop-ups" abre todas as vezes

### ISSUE THREAD

**Issue by A-Infor** [albertoleoncio/alberobot#25]
Foi muito boa a alteração que incluiu o botão "Avisar todos".
Mas a janela "Lembre-se de habilitar os pop-ups!" é um incômodo, pois é preciso habilitar os pop-ups apenas uma vez na vida, e a janelinha aparece TODAS as vezes que se usa a ferramenta.
Deveria ser possível que a janelinha salvasse um cookie para se lembrar que já me avisou e que já fiz a ação requerida por ela.

**Comment by albertoleoncio** [comment]
Acho mais prático usar [localStorage](https://www.w3schools.com/jsref/api_storage.asp), já que cookies envolvem outras dores de cabeça. Vou estudar como fazer isso, já que eu nunca mexi nessa função.

**Comment by A-Infor** [comment]
Claro, perfeito. Eu apenas sugeri da única forma que eu conhecia. Qualquer forma que funcione é perfeitamente válida.
